### PR TITLE
Fix: Complete Request - Contributors available across all communities…

### PR DESCRIPTION
**Pull request by lorenjohnson**
Allow all members of communities to which the Post belongs be selected as contributors to the request.

Requires related backend deploy: https://github.com/Hylozoic/hylo-node/pull/207

### CODE REVIEW

File: src/components/Post.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ export class Post extends React.Component {
 
   static contextTypes = {currentUser: object}
 
-  static childContextTypes = {post: object}
+  static childContextTypes = {post: object, dispatch: func, currentUser: object}
```
Comment (levity) [inline src/components/Post.js]: is this necessary? i would think those two are already available through childContextTypes in the top-level App component

File: test/client/components/Post.test.js [hunk 2]
```diff
@@ -151,68 +153,80 @@ describe('Post', () => {
...
-    node.find('.toggle').simulate('change')
-    expect(node.find('.done')).to.be.length(1)
-    node.find('.done').simulate('click')
-    expect(node.find('.contributors .person')).to.be.length(0)
-    expect(node.find('.contributors').text()).to.contain('completed')
-    expect(node.find('.toggle')).to.be.length(1)
-    mockify(window, 'confirm', prompt => true)
-    node.find('.toggle').simulate('change')
-    expect(node.find('.request-complete-heading').text()).to.contain('if this request has been completed')
+    it('requests contributors from all community associated to the post', () => {
```
Comment (levity) [inline test/client/components/Post.test.js]: s/community/communities/

File: src/components/Post.js (hunk 1 above)
Comment (lorenjohnson) [reply src/components/Post.js]: Surely not necessary at all, not sure why I ended-up adding that here. Correcting.

File: test/client/components/Post.test.js (hunk 2 above)
Comment (lorenjohnson) [reply test/client/components/Post.test.js]: Got it

**Comment by lorenjohnson** [comment]
I'm not understanding why CircleCI is failing the RichTextEditor tests. Everything passes locally. I tried a "Build without Cache" on Circle but same result.  Hmmm.

**Comment by levity** [comment]
Since the failure pattern is different across the two attempts it could just be timeouts related to, say, network lag in the VPS that CircleCI is using. Trying again.

**Pull request merged**